<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_docum_plan</class>
 <widget class="QWidget" name="Edit_docum_plan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="edc_name_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название работы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edc_name_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="edc_date_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Последнее изменение</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edc_date_edit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="edc_author_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cantarell</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QComboBox" name="edc_author_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="edc_save_button">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edc_cancel_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edc_delete_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edc_add_button">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>540</y>
     <width>88</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
